Makefile

```diff
@@ -724,15 +712,31 @@ release-prep:
 	bash releng/release-prep.sh
 
 release:
-	@if [ "$(VERSION)" = "$(GIT_VERSION)" ]; then \
-		printf "'make release' can only be run for a GA commit when VERSION is not the same as GIT_COMMIT!\n"; \
+	@if ! [[ '$(RELEASE_VERSION)' =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then \
+		printf "'make release' can only be run for commit tagged with 'vX.Y.Z'!\n"; \
 		exit 1; \
 	fi
-	docker pull $(AMBASSADOR_DOCKER_REPO):$(LATEST_RC)
-	docker tag $(AMBASSADOR_DOCKER_REPO):$(LATEST_RC) $(AMBASSADOR_DOCKER_REPO):$(VERSION)
-	docker push $(AMBASSADOR_DOCKER_REPO):$(VERSION)
+	docker pull $(AMBASSADOR_DOCKER_REPO):$(RELEASE_VERSION)-rc-latest
+	docker tag $(AMBASSADOR_DOCKER_REPO):$(RELEASE_VERSION)-rc-latest $(AMBASSADOR_DOCKER_REPO):$(RELEASE_VERSION)
+	docker push $(AMBASSADOR_DOCKER_REPO):$(RELEASE_VERSION)
 	$(MAKE) SCOUT_APP_KEY=app.json STABLE_TXT_KEY=stable.txt update-aws
 
+#release-ga: RETAG_INSTEAD_OF_REBUILD = true
+#release-ga: ambassadr.docker.push.release-ga
+#release-ga: SCOUT_APP_KEY = app.json
+#release-ga: STABLE_TXT_KEY = stable.txt
+#release-ga: update-aws
+
+release-rc: ambassadr.docker.push.release-rc
```

Comment:
In this, in `release-rc`, and in `release-ga`, are you intentionally dropping the 'o' in `ambassador` ?